example/advection/advection_package.cpp

```diff
@@ -194,9 +194,9 @@ Real EstimateTimestep(Container<Real> &rc) {
       pmb->pcoord->CenterWidth2(k, j, is, ie, dx2);
       pmb->pcoord->CenterWidth3(k, j, is, ie, dx3);
       for (int i = is; i <= ie; i++) {
-        min_dt = std::min(min_dt, dx1(i) / std::abs(vx));
-        min_dt = std::min(min_dt, dx2(i) / std::abs(vy));
-        min_dt = std::min(min_dt, dx3(i) / std::abs(vz));
+        if (vx != 0.0) min_dt = std::min(min_dt, dx1(i) / std::abs(vx));
+        if (vy != 0.0) min_dt = std::min(min_dt, dx2(i) / std::abs(vy));
+        if (vz != 0.0) min_dt = std::min(min_dt, dx3(i) / std::abs(vz));
```

Comment:
This is obviously an improvement over the previous code. However, the floating-point `!=` test strikes me as strange. Not sure there's a good solution, though. Checking for a very small velocity isn't right either.

Reply:
There's just one representation of 0 as far as I know and the division is only a problem if the value is actually 0, so I think this should be fine here.

Reply:
Yes. I agree. This has code-smell, but I think it's fine.